helm/adaptdl-sched/values.yaml

```diff
@@ -33,6 +33,12 @@ job:
 #    limits:
 #      cpu: "1.0"
 #      memory: "1Gi"
+  # Users can add customized patch for pods created by adaptdl.
+  # jobPatch:
+  #   # podPatch will be applied to pod level
+  #   podPatch: "[{\"op\": \"add\", \"path\": \"/metadata/annotations/k8s.v1.cni.cncf.io~1networks\", \"value\": \"macvlan-conf\"}]"
+  #   # containerPatch will be applied to each container
+  #   containerPatch: "[{\"op\": \"add\", \"path\": \"/env/0\", \"value\": {\"name\": \"NCCL_SOCKET_IFNAME\", \"value\": \"net1\"}}]"
```

Comment:
I think the example could just be written in-line as YAML, which makes writing this section a bit more readable, like
```
- op: add
  path: "/metadata/annotations/k8s.v1.cni.cncf.io~1networks"
  value: "macvlan-conf"
```